tests/RedisMutexTest.php

```diff
@@ -61,8 +62,8 @@ public function acquireTimeoutProvider()
     }
 
     /**
-     * @covers \yii\redis\Mutex::acquireLock
-     * @covers \yii\redis\Mutex::releaseLock
+     * @covers       \yii\redis\Mutex::acquireLock
+     * @covers       \yii\redis\Mutex::releaseLock
```

Comment:
Same here. Why so many spaces?

Reply:
some strange. I don't can change it in rules =-_-=